benchmarks/common.py

```diff
@@ -1257,139 +1257,6 @@ def parse_args():
         "--quiet", "-q", action="store_true", help="suppress debug printouts"
     )
 
-    group = parser.add_mutually_exclusive_group()
```

Comment:
The main downside of remove this is we lose our `--help` documentation.  We may need to manually generate `--help` if we go this route.

Reply:
Does anyone... actually read the `torchbench.py --help` documentation though? There's so many flags, it seems hard for someone to find what they're looking for

Reply:
I use `--help` when I forget a flag name.